### DISCUSSION TITLE
Hiding the posts details table on each post

### DISCUSSION THREAD

**Discussion by puneet-gupta** [mmistakes/minimal-mistakes#2578] [Q&A] [answered]
I am noticing a table categorized by the year of posts on each blog post
For e.g. on https://puneet-gupta.github.io/puneetgupta/2020/06/01/collect-network-trace-via-AutoHealing/

![image](https://user-images.githubusercontent.com/8878114/85295602-cb3fdc00-b4bd-11ea-85dd-c7f8b30c563b.png)

I want to display only the POST content when someone goes to the post and avoid showing this table and list of posts on every post. Is there a way to workaround this?

**Comment by iBug** [comment, marked as answer]
Get rid of this thing in Front Matter:

> ![image](https://user-images.githubusercontent.com/7273074/85295982-4cee3480-b4d3-11ea-8546-9db291e0ba94.png)

**Comment by puneet-gupta** [comment]
Super! That was exactly the problem. Thanks again!
